docs/update_castxml_version.rst

```diff
@@ -6,9 +6,43 @@ Updating the CastXML version
 
 A developer should use the following steps to update the version ``X.Y.Z``
 of CastXML associated with the current CastXML python distributions.
-
 Available CastXML archives can be found `here <https://data.kitware.com/#folder/57b5de948d777f10f2696370>`_.
 
+First, build the artifacts which will be made available for download.
+
+Update CastXML files
```

Comment:
Let's move this directly into https://github.com/CastXML/CastXMLSuperbuild/blob/master/README.rst adding a section like 

```
Maintainers
-----------

* `How to update CastXML version ? <https://github.com/CastXML/CastXMLSuperbuild/blob/master/docs/make_a_release.rst>`_
* `How to make a release ? <https://github.com/CastXML/CastXMLSuperbuild/blob/master/docs/update_castxml_version.rst>`_
```